### DISCUSSION TITLE
How to export only displayed columns?

### DISCUSSION THREAD

**Discussion by DanPresa** [KevinVandy/material-react-table#398] [Q&A]
I have a table but. I only need to be able to export displayed columns except for the hide ones.
Is it possible to do that?

**Comment by KevinVandy** [comment]
`table.getRowModel().rows.map(row => row.getVisibleCells()).map(cell => cell.getValue)` or something like that

**Comment by d-zheng** [comment]
I went with this:
```
const rowData = table.getRowModel().rows.map(row => { const r = {};  row.getVisibleCells().map(cell => { r[cell.column.id] = cell.getValue() }); return r; });
```
